CHANGELOG.md

```diff
@@ -14,11 +14,11 @@
 
 ## Compatibility Notes
 
--
+- Importing filter functions directly is deprecated, import using `filterFunctions` instead.
```

Comment:
I do not see any change in index.ts or imports indicating this change? I see in the tests that it is there though. Could you enlighten me and tell me how this is done.

Reply:
Yeah, I didn't know how to deprecate that. For backwards compatibility importing `length` directly is still possible, but in the future this will be removed. (This happened on the last PR, btw.)

Reply:
Ah ok, I also now found the lines in the `indes.ts` files: `export * as filterFunctions from './filter-functions';`. So this was already in the #227 commit.